applications/nrf5340_audio/src/bluetooth/bt_content_control/bt_content_ctrl.h

```diff
@@ -50,14 +50,11 @@ int bt_content_ctrl_discover(struct bt_conn *conn);
  *
  * @note	This partial data is used to build a complete extended advertising packet.
  *
- * @param [in/out] uuid_buf Buffer being populated with UUIDs.
- * @param [in/out] adv_buf Buffer being populated with ext adv elements.
- * @param [in] adv_buf_vacant Number of vacant elements in @p adv_buf.
+ * @param[out]	uuid_buf	Buffer being populated with UUIDs.
  *
  * @return	Negative values for errors or num elements added to @p adv_buf.
```

Comment:
```suggestion
 * @return	0 for success, error otherwise.
```

This is only if we remove adv_buf and adv_buf_vacant. It was intentional that all _adv_get functions have the same interface, even if adv_buf and adv_buf_vacant is not used.
Same for all _adv_get functions.